### PR TITLE
Edits to postgresql uninstall topic pr5055

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

**Review by djw-m — CHANGES_REQUESTED** [review]
You seem to have removed all the images from the uninstallation page.

**Review by dwicinas — DISMISSED** [review]

File: advocacy_docs/supported-open-source/postgresql/uninstalling.mdx [hunk 1]
```diff
@@ -19,29 +19,19 @@ The PostgreSQL installer creates an uninstaller in the PostgreSQL installation d
 
 ## Uninstalling PostgreSQL on Windows
 
-1.  On the Windows Control Panel, open the Uninstall or change a program dialog box. 
-
-  ![The Uninstall or change a program dialog box](/installing/images/change_program.png)
+1.  On the Windows Control Panel, open the Uninstall or Change a Program dialog box.
```
Comment (ebgitelman) [inline advocacy_docs/supported-open-source/postgresql/uninstalling.mdx:22]:
```suggestion
1.  On the Windows Control Panel, open the Uninstall or Change a Program dialog box. 

![The Uninstall or Change a Program dialog box](/installing/images/change_program.png)
```

File: advocacy_docs/supported-open-source/postgresql/uninstalling.mdx [hunk 2]
```diff
@@ -19,29 +19,21 @@ The PostgreSQL installer creates an uninstaller in the PostgreSQL installation d
 
 ## Uninstalling PostgreSQL on Windows
 
-1.  On the Windows Control Panel, open the Uninstall or change a program dialog box. 
+1.  On the Windows Control Panel, open the Uninstall or Change a Program dialog box. 
 
-  ![The Uninstall or change a program dialog box](/installing/images/change_program.png)
+![The Uninstall or Change a Program dialog box](/installing/images/change_program.png)
 
 1. Right-click **PostgreSQL**, and select **Uninstall/Change** from the context menu.
```
Comment (ebgitelman) [inline advocacy_docs/supported-open-source/postgresql/uninstalling.mdx:26]:
```suggestion
1. Right-click **PostgreSQL**, and select **Uninstall/Change** from the context menu.

![Confirm that you want to uninstall PostgreSQL](/installing/images/uninstalling_postgreSQL_comfirmation.png)
```

File: advocacy_docs/supported-open-source/postgresql/uninstalling.mdx [hunk 3]
```diff
@@ -19,29 +19,23 @@ The PostgreSQL installer creates an uninstaller in the PostgreSQL installation d
...
 
-  ![Confirm that you want to uninstall PostgreSQL](/installing/images/uninstalling_postgreSQL_comfirmation.png)
+![Confirm that you want to uninstall PostgreSQL](/installing/images/uninstalling_postgreSQL_comfirmation.png)
 
 1.  You can remove the entire application or individual components.
 
     - To remove the entire application, select **Entire application** and select **Next**.
     - To remove individual components, select **Individual components** and select **Next**. Then, from the list, select the components to uninstall, and select **Next**.
```
Comment (ebgitelman) [inline advocacy_docs/supported-open-source/postgresql/uninstalling.mdx:33]:
```suggestion
    - To remove individual components, select **Individual components** and select **Next**. Then, from the list, select the components to uninstall, and select **Next**.
    
    ![Select the components to uninstall](/installing/images/select_the_components_to_uninstall.png)
```

File: advocacy_docs/supported-open-source/postgresql/uninstalling.mdx [hunk 4]
```diff
@@ -19,29 +19,25 @@ The PostgreSQL installer creates an uninstaller in the PostgreSQL installation d
...
-
-  ![Select the components to uninstall](/installing/images/select_the_components_to_uninstall.png)
+    
+    ![Select the components to uninstall](/installing/images/select_the_components_to_uninstall.png)
 
   A progress bar keeps you informed as PostgreSQL is removed.
```
Comment (ebgitelman) [inline advocacy_docs/supported-open-source/postgresql/uninstalling.mdx:37]:
```suggestion
  A progress bar keeps you informed as PostgreSQL is removed.
  
  ![Uninstalling PostgreSQL](/installing/images/uninstalling_postgreSQL.png)
```

File: advocacy_docs/supported-open-source/postgresql/uninstalling.mdx [hunk 5]
```diff
@@ -19,29 +19,27 @@ The PostgreSQL installer creates an uninstaller in the PostgreSQL installation d
...
-
+  
   ![Uninstalling PostgreSQL](/installing/images/uninstalling_postgreSQL.png)
 
 1.  When the uninstaller completes, select **OK**.
```
Comment (ebgitelman) [inline advocacy_docs/supported-open-source/postgresql/uninstalling.mdx:41]:
```suggestion
1.  When the uninstaller completes, select **OK**.

![An Info dialog box confirms the uninstallation](/installing/images/uninstallationcomplete.png)
```

File: advocacy_docs/supported-open-source/postgresql/uninstalling.mdx [hunk 6]
```diff
@@ -71,19 +71,11 @@ confirming what it has not been removed at the end of its run.
 
   The uninstaller opens and asks if you want to uninstall the entire application or individual components.
```
Comment (ebgitelman) [inline advocacy_docs/supported-open-source/postgresql/uninstalling.mdx:72]:
```suggestion
  The uninstaller opens and asks if you want to uninstall the entire application or individual components. 
  
  ![Remove the entire application or components](/installing/images/remove_the_entire_application_or_components.png)
```

File: advocacy_docs/supported-open-source/postgresql/uninstalling.mdx [hunk 7]
```diff
@@ -70,20 +70,14 @@ confirming what it has not been removed at the end of its run.
...
 
   The uninstaller opens and asks if you want to uninstall the entire application or individual components. 
-
+  
   ![Remove the entire application or components](/installing/images/remove_the_entire_application_or_components.png)
 
 1.  Respond to the installer:
 
     -  To remove the entire application, select **Entire application**, and select **Next**.
     -  To remove individual components, select **Individual components**. Select the components to uninstall, and select **Next** to start uninstalling components.
```
Comment (ebgitelman) [inline advocacy_docs/supported-open-source/postgresql/uninstalling.mdx:79]:
```suggestion
    -  To remove individual components, select **Individual components**. Select the components to uninstall, and select **Next** to start uninstalling components.
    
![Selecting the components to uninstall](/installing/images/selecting_the_components_to_uninstall.png)
```

File: advocacy_docs/supported-open-source/postgresql/uninstalling.mdx [hunk 8]
```diff
@@ -70,20 +70,16 @@ confirming what it has not been removed at the end of its run.
...
-
-  ![Selecting the components to uninstall](/installing/images/selecting_the_components_to_uninstall.png)
+    
+![Selecting the components to uninstall](/installing/images/selecting_the_components_to_uninstall.png)
 
   Progress bars appear as each component is removed. 
 
 1.  When the uninstaller completes, select **OK**.
```
Comment (ebgitelman) [inline advocacy_docs/supported-open-source/postgresql/uninstalling.mdx:85]:
```suggestion
1.  When the uninstaller completes, select **OK**.

![The Uninstallation is completed](/installing/images/mac_uninstallation_completed.png)

```

**Comment by dwicinas** [comment]
@ebgitelman The step numbers are not working correctly. Probably has to do with how the re-inserted graphics are positioned in the text hierarchy.

**Pull request merged**